libs/designer-ui/src/lib/arrayeditor/collapsedarray.tsx

```diff
@@ -23,7 +23,6 @@ export interface CollapsedArrayProps {
 
 export const CollapsedArray = ({
   labelProps,
-  isValid = true,
```

Comment:
here we were doing validation checks within the editor as well as the parameter validation to enable/disable the toggle button. Removed this so there would never be the case of 2 error messages showing.